src/utils/util.js

```diff
@@ -97,6 +107,18 @@ function isPluginUrl(serverUrl, inputURL) {
       parsedURL.pathname.toLowerCase().startsWith('/plugins/')));
 }
 
+function isTrustedRemoteUrl(serverUrl, inputURL) {
+  const server = getServerInfo(serverUrl);
+  const parsedURL = parseURL(inputURL);
+  if (!parsedURL || !server) {
+    return false;
+  }
+  return (
+    equalUrlsIgnoringSubpath(server, parsedURL) &&
+    (parsedURL.pathname.toLowerCase().startsWith(`${server.subpath}trusted/`) ||
+      parsedURL.pathname.toLowerCase().startsWith('/trusted/')));
```

Comment:
I still don't feel like forcing all the users config to match your setup, so we should probably aim for something more configurable, this is just a suggestion on how it would look like:

```suggestion
  const trustedPaths = getTrustedPaths(); // this function would ideally look into some config (like buildConfig.js) to add `trusted` or any other paths we want to add. It also allows for configuring several of them if needed.
  return (
    equalUrlsIgnoringSubpath(server, parsedURL) && trustedPaths && trustedPaths.lenght &&
    trustedPaths.some((trustPath) => (parsedURL.pathname.toLowerCase().startsWith(`${server.subpath}${trustPath}/`) || parsedURL.pathname.toLowerCase().startsWith('/${trustPath}/'))));
```

**Note:** `getTrustedPaths` is not defined anywhere